### ISSUE TITLE
Maintainer

### ISSUE THREAD

**Issue by antillas21** [jbox-web/ajax-datatables-rails#17]
Hi @JoelQ ,

I would like to propose to take over maintenance for this project.

I have some specific ideas on features to add to it. Recently I forked a repo and [added some of these ideas there](https://github.com/antillas21/rails-datatables/commit/7097f7854a319c21c33c5e77bb1cf0cbff4abc60).

In the end I discovered, that repo is a fork of yours, LOL.

Looking forward to your comments.

**Comment by JoelQ** [comment]
I'm a fan of your changes, particularly adding tests and a namespace. This was the first gem I wrote and the code could really use some cleaning up :smiley: 

Shall I transfer the repo to your account?

**Comment by antillas21** [comment]
Glad you liked my proposed changes :)

I'll be flattered if you transfer ownership of this repo to me.

**Comment by JoelQ** [comment]
Github won't allow me to transfer the repo because you already have a fork of it.

**Comment by antillas21** [comment]
Let me destroy that fork... I still have a local copy, so changes won't be
lost.

**Comment by antillas21** [comment]
@JoelQ done... try again, please.

**Comment by JoelQ** [comment]
Done. I look forward to seeing where you take this project.

**Comment by antillas21** [comment]
Excellent!

I just accepted the transfer :smile:... 

I think there's some change that needs to be done so I can push to Rubygems, too? or would this be automatically when I run `$ rake release`?

**Comment by JoelQ** [comment]
I'll need the email you use for your rubygems account so I can add it as an owner on rubygems

**Comment by antillas21** [comment]
I see... email address is: antillas21@gmail.com

Feeling excited about these changes!

**Comment by JoelQ** [comment]
Added you as an owner on rubygems. You should now be able to push new versions.

**Comment by antillas21** [comment]
Thanks @JoelQ !

I will notify you when these changes are released (or maybe Github & Rubygems will notify you first, LOL).

Here's to make you proud of the project you started.

**Comment by antillas21** [comment]
@JoelQ 

Just to let you know that I have release a new version (v. 0.1.0).

I have included some major changes into modules and classes organization... perhaps you would like to know some of the future plans for this gem: [Support for other ORMs](https://github.com/antillas21/ajax-datatables-rails#orm-support)

I have updated the README... the most important changes come from this [Additional Notes](https://github.com/antillas21/ajax-datatables-rails#additional-notes) section, and from the fact that all search queries are now "database agnostic" as long as the Rails app relies on `ActiveRecord`.
